flixel/system/debug/interaction/tools/Pointer.hx

```diff
@@ -21,13 +21,17 @@ class GraphicCursorCross extends BitmapData {}
  */
 class Pointer extends Tool
 {
+	public var _selectionArea:FlxRect = new FlxRect();
```

Comment:
Why was this made public? your not using it in any external classes

Reply:
My bad, I forgot to remove it after testing something